handbook/project-management/scope.md

```diff
@@ -36,13 +36,24 @@ This timeline is a *new and proposed* timeline, and will probably change based o
 :::
 
 * **Week 0**: Brainstorm ideas and start to consolidate around one or two ideas
-  * Designers will focus on user testing and prototyping
+  * Designers will focus on user research
+    * This will involve activities like ideation sessions, interviews, personas, journey maps, user stories, sketching & heat-  map voting.
+    * Initial ideation will involve the whole team (coming up with ideas & possible features)
+    * Designers may also ask developers to help with interviews and surveys to get more data.
 * **Week 2**: Finalized project pitch and MVP writeup
-  * Designers will continue user testing and prototyping with a focus on the MVP
+  * Designers will start wireframing and prototyping with a focus on the MVP
+    * Week 2: Mid-Fi Wireframes & prototype should be done by the end of the week.
+    * Week 3: User testing on Mid-fi Prototype.
+    * Week 4: Hi-Fi wireframes & Prototype should be done.
+    * At any point, always upload your finished designs (Mid-Fi & Hi-Fi) to your centralized drive for developers to access and always alert them when you have done so.
   * Developers will start implementation work
 * **Week 5**: A *functional* MVP based on initial idea and prototypes
   * Teams should start asking people to try initial implementation and give ideas and feedback
   * Designers should start work on collecting feedback based on the implementation, branding, and improving the MVP
+    * Week 5: Continue user testing with the MVPs & may start playing around with microinteractions, branding.
+    * Week 6: Start with branding (Logos, Name, Colours, UI Library), keep developers part of your branding ideation (let them do voting, this is their project as much as it is yours!)
+    * Week 7: Finalize branding, name, logo etc. 
```

Comment:
I'm concerned with giving *too* much direction - I imagine projects will have varying needs and thus progress at too different of a pace to give detailed week-by-week guidance (which is why the original guidance tried to give high-level overviews of milestones to hit at 2-3 week intervals)

If you feel strongly about it though, I'll adjust the list a little to make it clear that this is design-specific guidance?

Reply:
Mhm yes, I see what you mean! I intended it to be more of a guideline that they don't have to follow. Usually, we go off track when needed (like exams, or other things). In my experience, it's quite daunting to start on a project without having a specific guideline you can follow at first. I think that outlining this for designers will help them visualize where they need to be at a certain point in the sprint!

And yes, this is solely for designers! I understand that developers will probably have a lot more varying paces, I just want to make sure that design is able to finish everything they need to so that they can give time for developers to code and implement.

Reply:
Great! I've cleaned up the formatting to help make it clearer what's expected of each role, and make it clear these points are specifically for design :)